user_pinger.py

```diff
@@ -448,6 +464,32 @@ def list_groups(_, author: praw.models.Redditor) -> None:
             self._send_pm("Avaliable Groups", [groups_list], author)
             return
 
+        def list_my_groups(_, author: praw.models.Redditor) -> None:
+            """
+            Returns all the groups the user is a part of
+
+            Usage:
+            body = listmygroups
+            """
+            self.logger.debug("Getting groups")
+            groups: ConfigParser = self._get_wiki_page(["config", "groups"])
+            self.logger.debug("Got groups")
+
+            self.logger.debug("Getting User's groups")
+            myGroups = []
+
+            for (group_name, usernames) in groups.items():
+                for username in usernames.items():
+                    username = str(username[0]).upper()
+                    if str(author).upper() == username.upper():
+                        myGroups.append(group_name);
+
+            groups_list = ', '.join(myGroups)
+            self.logger.debug("Found User's groups")
+
+            self._send_pm("Groups you are in", [groups_list], author)
```

Comment:
"Hello [username]! You're currently subscribed to these ping groups: [groups_list]." or something friendly like that seems better to me.